PowerUp/app/src/main/res/values/strings.xml

```diff
@@ -108,7 +108,4 @@
     <string name="memory_label">Does the tile on the left match the tile that appeared one tile before it?</string>
     <string name="no">NO</string>
     <string name="yes">YES</string>
-    <string name="tutorial_1">Shoot the viruses with the syringe!</string>
-    <string name="tutorial_2">The color of the syringe should match with the color of the virus otherwise you loose!</string>
-    <string name="tutorial_3">Try to kill as many viruses as possible before the time runs out!</string>
 </resources>
```

Comment:
You have used these 3 fields in activity_kill_the_virus_tutorial.xml file. It will give errors if you remove them here.